corehq/apps/domain/models.py

```diff
@@ -644,14 +644,6 @@ def generate_name(cls, hr_name, max_length=25):
 
         return name
 
-
-    def password_format(self):
-        """
-        This was a performance hit, so for now we'll just return 'a' no matter what
-        If a single application is alphanumeric, return alphanumeric; otherwise, return numeric
-        """
-        return 'a'
```

Comment:
I think the intention here was to eventually restore that behavior, but it'd been there for 3 years, and if we DO restore it, we should do it in a better way.  Anyways, that's a single commit, so it can be reverted if needed.